changelog/@unreleased/pr-961.v2.yml

```diff
@@ -0,0 +1,9 @@
+type: fix
+fix:
+  description: |-
+    Rewrite all tests to use undertow...not dropwizard
+
+    This lets us then upgrade to Jakarta without also upgrading
+    slf4j versions
+  links:
+  - https://github.com/palantir/tracing-java/pull/961
```

Comment:
I probably wouldn't have included a changelog with this PR since it doesn't have any impact on sonsumers

Reply:
ya, I wasn't sure how releases work in this repository and if people read the release notes that well.